SearchForm.js

```diff
@@ -0,0 +1,117 @@
+const baseUrl =
+  "https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/";
+
+class SearchForm {
+  constructor(formElement) {
+    this.formElement = formElement;
+    this.userInput = this.formElement.querySelector(".s-text-input");
+    this.searchBtn = this.formElement.querySelector(".s-btn");
+    this.ul = document.querySelector('ul');
+    this.formURL = `${baseUrl}search?query=`;
+    this.debounceTimer;
+    this.InputEventListener();
+  }
+  //Event Listeners
+  InputEventListener() {
+    this.userInput.addEventListener("keyup", (e) => {
```

Comment:
do you really need this event listener, or could the if statement be inside the eventlistener of the "input" event?